packages/suite-desktop-core/e2e/tests/manual/eth/eth-transactions-NFT.test.ts

```diff
@@ -0,0 +1,66 @@
+import { TestCategory, TestPriority, TestStream } from '../../../support/enums/testAnnotations';
+import { test } from '../../../support/fixtures';
+import { createTestAnnotation } from '../../../support/reporters/annotations';
+
+// Transaction types
+// Send NFT and check change in Suite
+
+test.describe.skip('Eth transactions NFT', { tag: ['@group=manual'] }, () => {
```

Comment:
BSC uses BEP standard, not ERC, would be nice to cover that as well (1 type is enough), easy to get NFT is ENS